src/controller/getAllUrl.ts

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+import { databaseModel } from "../model";
+import { logger } from "../services";
+
+export const getAllUrl = async (
+  req: express.Request,
+  res: express.Response,
```

Comment:
I prefer just using the destructured version. Apply this to every necessary
```suggestion
  _: Request,
  res: Response,
```